index.bs

```diff
@@ -1510,23 +1511,36 @@ enum AudioContextLatencyCategory {
 </table>
 </div>
 
-<xmp class="idl">
-[Exposed=Window]
-interface AudioContext : BaseAudioContext {
-	constructor (optional AudioContextOptions contextOptions = {});
-	readonly attribute double baseLatency;
-	readonly attribute double outputLatency;
-	AudioTimestamp getOutputTimestamp ();
-	Promise<undefined> resume ();
-	Promise<undefined> suspend ();
-	Promise<undefined> close ();
-	MediaElementAudioSourceNode createMediaElementSource (HTMLMediaElement mediaElement);
-	MediaStreamAudioSourceNode createMediaStreamSource (MediaStream mediaStream);
-	MediaStreamTrackAudioSourceNode createMediaStreamTrackSource (
-		MediaStreamTrack mediaStreamTrack);
-	MediaStreamAudioDestinationNode createMediaStreamDestination ();
-};
-</xmp>
+<div class="correction proposed" id="c2444-1">
+	<span class="marker">Proposed Addition
+		<a href="https://github.com/WebAudio/web-audio-api/issues/2444">Issue
+			2444</a>.
+	</span>
+	Add AudioRenderCapacity Interface
+	<div class="amendment-buttons">
+		Buttons here
+	</div>
+	<ins cite=#c2444>
```

Comment:
@padenot We have a technical problem here - it seems like having `ins` and `del` doesn't work within the scope of `xmp`. This was the only way to make it pass the build check. Please advise if you have a better idea to handle this.

Reply:
This is https://github.com/tabatkins/bikeshed/issues/2109, filed by Raymond a while back. Suggested workaround is https://github.com/tabatkins/bikeshed/issues/2109#issuecomment-931612851.

Reply:
Hmm. I actually tried that method in previous commit, but then bikeshed complained about duplicate IDL definition. I'll try again.

Reply:
Huh, I used `data-no-idl` to disable IDL parsing on the previous IDL definition. It worked. :)

Reply:
> Huh, I used `data-no-idl` to disable IDL parsing on the previous IDL definition. It worked. :)

... but Bikeshed still adds the block to the final IDL index, so duplication still exists for extraction tools that process the index. See #2486 for a possible fix.